app/src/main/java/com/example/soundcloud/selected_genre_detail/SelectedGenreActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.example.soundcloud.selected_genre_detail;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.example.soundcloud.R;
+
+public class SelectedGenreActivity extends AppCompatActivity {
+    public static final String GENERE_KEY = "GENERE_KEY";
+
+    public static Intent getIntent(Context context) {
```

Comment:
a cần pass dữ liệu gì thì có thể pass qua method này luoon
```suggestion
    public static Intent getIntent(Context context, .......) {
```